source/stylesheets/_refills-header.scss

```diff
@@ -0,0 +1,100 @@
+.refills-header {
+  @include clearfix;
+  background-color: $refills-hero-background;
+  clear: both;
+  margin-bottom: 1em;
+  padding: 5em 0;
+  text-align: center;
+  width: 100%;
+
+  @include media($refills-medium-screen) {
+    margin-bottom: 4em;
+    padding: 6em 0;
+  }
+
+  @include media($refills-large-screen) {
+    margin-bottom: 4em;
+    padding: 6em 0 11em 0;
+  }
+
+  .refills-logo {
+    display: inline-block;
+    margin: 0 auto 1.4em auto;
+
+    @include media($refills-medium-screen) {
+      max-width: 8em;
+      max-height: 8em;
+    }
+
+    img {
+      background: #fff;
+      border-radius: 50%;
+      padding: 0.3em;
```

Comment:
Properties should be ordered background, border-radius, margin-bottom, max-height, max-width, padding